PrismMockProvider/auth/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM public.ecr.aws/lambda/python:3.10
+
+COPY src/ ${LAMBDA_TASK_ROOT}
```

Comment:
Where is this var is set? Can't we just hardcode a path? it's in an image so any path should just work i think.

Reply:
`LAMBDA_TASK_ROOT` is a reserved environment variable containing the path to the Lambda function's code, this is set by the lambda runtime during initialization.

See the docs on this [here](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime)